projects/packages/jitm/src/class-jitm.php

```diff
@@ -44,11 +53,28 @@ public static function get_instance() {
 	}
 
 	/**
-	 * Determines if JITMs are enabled.
+	 * Sets up JITM action callbacks if needed.
+	 */
+	private function register() {
```

Comment:
Should we be concerned that the previously `public` register method now became `private`?

I didn't find any calls outside of the package itself, but raising awareness just to be sure!

Reply:
Good question! Originally I wasn't concerned since that method isn't called outside of the package. But we have called that method outside of the package in the past. [This PR](https://github.com/Automattic/jetpack/pull/14424) from January 2020 is where we started using the `JITM::configure` method instead of `register`.

I'll change the visibility of the `register` method back to public. Honestly, I'm not sure how we could end up in a situation where we have an old version of Jetpack that calls `JITM::register` and a new version of the JITM package with the private `register` method. But, I'd rather not take the risk.

Reply:
I switched the method back to `public` in 12a0c18. I also moved the check of the `self::$jitms_registered` property into the `register` method.